proof/crefine/AARCH64/Fastpath_Equiv.thy

```diff
@@ -557,7 +557,7 @@ lemma zipWithM_setRegister_simple_modify_registers:
 
 (* FIXME AARCH64 move to IsolatedThreadAction where existing setRegister_simple is commented out *)
 lemma setRegister_simple:
-  "setRegister r v = (\<lambda>con. ({((), UserContext (fpu_state con) ((user_regs con)(r := v)))}, False))"
+  "setRegister r v = (\<lambda>con. ({((), UserContext (user_fpu_state con) ((user_regs con)(r := v)))}, False))"
```

Comment:
Not really related to this PR, but it looks like `setRegister_simple` has already been uncommented in `IsolatedThreadAction`, so this copy of it can probably be removed.